README.md

```diff
@@ -1,3 +1,109 @@
+<!-- Add banner here -->
+![banner](https://imgur.com/QwALmAn.png)
 # Kubiquity
 
+<!-- Add buttons here -->
+![Version](https://img.shields.io/badge/version-1.0.0-blue.svg?cacheSeconds=2592000)
+![GitHub release (latest by date including pre-releases)](https://img.shields.io/github/v/release/oslabs-beta/Kubiquity?include_prereleases)
+![GitHub pull requests](https://img.shields.io/github/issues-pr/oslabs-beta/kubiquity)
+[![Documentation](https://img.shields.io/badge/documentation-yes-brightgreen.svg)](https://github.com/oslabs-beta/Kubiquity#readme)
+[![License: MIT](https://img.shields.io/github/license/oslabs-beta/Kubiquity)](https://github.com/oslabs-beta/Kubiquity/blob/master/LICENSE)
+[![Twitter: kubiquityapp](https://img.shields.io/twitter/follow/kubiquityapp.svg?style=social)](https://twitter.com/kubiquityapp)
 
+
+### 🏠 [Homepage](https://kubiquity)
+
+>Kubiquity is a real time Kubernetes error monitoring tool that incorporates Prometheus metrics queries for CPU and memory usage that runs locally as an Electron application.
+
+
+
+# Preview
+
+![Demo](https://imgur.com/BHQuJVW.gif)
+# Table of contents
+
+- [Kubiquity](#kubiquity)
+    - [🏠 Homepage](#-homepage)
+- [Preview](#preview)
+- [Table of contents](#table-of-contents)
+- [Installation](#installation)
+- [Usage](#usage)
+- [TODO](#todo)
+- [Contributions](#contributions)
+- [Contributors](#contributors)
+- [License](#license)
+
+# Installation
+[(Back to top)](#table-of-contents)
+
+Visit our [releases page](https://github.com/oslabs-beta/Kubiquity/releases) to download the latest stable release for your operating system
+
+To package the app from the source code, fork and clone the repo, then run the following commands
+
+1. ```yarn```
+2. ```yarn run package-linux``` or ```yarn run package-win``` based on your OS"
+3. The app will be created and placed in the ```release-builds``` folder
+
+# Usage
+[(Back to top)](#table-of-contents)
+
+- Ensure that a Kubernetes cluster is running.
+
+- Create a Prometheus namespace in the cluster and install Prometheus into that namespace.
+
+- When running the app, ensure that Prometheus has been running for at least a minute so that it can generate metrics from the cluster.
+
+
+# TODO
```

Comment:
Future iterations or future implementations or planned implementations for header name, not todo.